locationserver/README.md

```diff
@@ -0,0 +1,60 @@
+# Location Server for nParse
+
+
+
+The location server runs on port: `8424`
+
+Set your nParse `Setttings -> Sharing -> Sharing Hostname` to: `ws://localhost:8424` (You can replace `locahost` with your ip or domain name)
+
+Running
+========
+
+Setup:
+
+1. Create the directory where you would like nParse to be cloned to and then `cd` into the directory.
+2. Clone the repository: `git clone https://github.com/nomns/nparse.git .`
+3. Create a virtual environment: `python -m venv .venv`
```

Comment:
This is technically wrong as it's designed to be run in a docker container 😅
But also I think I'll possibly need to make some modifications to the dockerfile too, so I can fix this later.